lib/modules/manager/crossplane/readme.md

```diff
@@ -1,40 +1,39 @@
-The `crossplane` manager has no `fileMatch` default patterns, so it won't match any files until you configure it with a pattern.
-This is because there is no commonly accepted file/directory naming convention for crossplane YAML files and we don't want to check every single `*.yaml` file in repositories just in case any of them have Crossplane Packages definitions: Configurations, Providers, Functions.
+To use the `crossplane` manager you must set your own `fileMatch` pattern.
+The `crossplane` manager has no default `fileMatch` pattern, because there is no common filename or directory name convention for Crossplane YAML files.
+By setting your own `fileMatch` Renovate avoids having to check each `*.yaml` file in a repository for a Crossplane Package definition.
 
-If most `.yaml` files in your repository are Crossplane ones, then you could add this to your config:
+The `crossplane` manager supports these `depType`s:
 
-```json
+- `configuration`
+- `function`
+- `provider`
+
+You can use these `depType`'s to control which dependencies Renovate will upgrade.
+
+If you need to change the versioning format, read the [versioning](../../../modules/versioning.md) documentation to learn more.
+
+Some configuration examples:
+
+```json title="fileMatch if most .yaml files are for Crossplane"
```

Comment:
The same question about "What if there are other `.yaml` files in the repository" applies here. 😉

Reply:
it would just try to read them and ignore them if they don't contain the right resources. Not the best for performance, but we tried to contain it by shortcircuiting if the file doesn't contain any match for the needed ApiVersion, so no big issue if people just set this

Reply:
I'll let `@rarkins` decide about the code change you're proposing. I don't know if trying to read the files and ignoring them is okay or not. 😄

Reply:
No code change, that's already how it's implemented.